backend/__tests__/server.test.js

```
@@ -0,0 +1,566 @@
+    });
+    
+    test('should exit if MONGO_URI is missing', () => {
+      // Save original
+      const originalUri = process.env.MONGO_URI;
+      const originalExit = process.exit;
+      
+      // Mock process.exit
+      process.exit = jest.fn();
+      
+      // Remove MONGO_URI
+      delete process.env.MONGO_URI;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)